src/components/appMenuSections/profile/CommonInfoProfile.tsx

```diff
@@ -9,10 +9,28 @@ import { network, getExplorerUrl } from '../../../consts/network';
 const CommonInfoProfile = () => {
   const currentRole = useAppSelector(selectCurrentUserRole);
   const [connectedWallet, setConnectedWallet] = useState<string | null>(null);
+  const [userAddress, setUserAddress] = useState<string | null>(null);
+  const [explorerLink, setExplorerLink] = useState<string | undefined>(undefined);
   const { currentTheme } = useCurrentTheme();
-  const explorerLink = getExplorerUrl[network]().explorerUrl;
+  // const explorerLink = getExplorerUrl[network](userAddress).explorerUrl;
   const userSession = useAppSelector(selectUserSessionState);
 
+  useEffect(() => {
+    if (userSession.isUserSignedIn()) {
+      const args = userSession.loadUserData().profile.stxAddress.testnet;
```

Comment:
it should be modular testnet/mainnt 

tesnet for devnet/testnet
mainnet for mainnet

Reply:
https://github.com/stacks-degens/starters-front-end/issues/46